.gitignore

```diff
@@ -9,6 +9,7 @@ geopandas/version.py
 doc/_static/world_*
 examples/nybb_*.zip
 .cache/
+*.c
```

Comment:
the thing is that now vectorized.c is in the list of untracked file, I can of course also add `geopandas/vectorized.c` explicitly here in .gitignore